src/routes/console/project-[project]/auth/security/updateMockNumbers.svelte

```diff
@@ -0,0 +1,128 @@
+<script lang="ts">
+    import { Submit, trackError, trackEvent } from '$lib/actions/analytics';
+    import { CardGrid, Heading } from '$lib/components';
+    import { InputPhone, InputText } from '$lib/elements/forms';
+    import { Button, Form, FormItem, FormItemPart } from '$lib/elements/forms';
+    import type { MockNumber } from '$lib/sdk/auth';
+    import { sdk } from '$lib/stores/sdk';
+    import { project } from '../../store';
+    import { addNotification } from '$lib/stores/notifications';
+    import { invalidate } from '$app/navigation';
+    import { Dependencies } from '$lib/constants';
+    import { writable } from 'svelte/store';
+    import Empty from '$lib/components/empty.svelte';
+
+    let numbers = writable<MockNumber[]>($project.authMockNumbers);
+    let initialNumbers = [];
+    let projectId: string = $project.$id;
+    let submitDisabled = true;
+
+    $: initialNumbers = $project.authMockNumbers.map((num) => ({ ...num }));
+    $: submitDisabled = JSON.stringify($numbers) === JSON.stringify(initialNumbers);
+
+    async function updateMockNumbers() {
+        try {
+            await sdk.forConsole.auth.updateMockNumbers(projectId, $numbers);
+            await invalidate(Dependencies.PROJECT);
+            addNotification({
+                type: 'success',
+                message: 'Updated mock phone numbers successfully'
+            });
+            trackEvent(Submit.AuthMockNumbersUpdate);
+        } catch (error) {
+            addNotification({
+                type: 'error',
+                message: error.message
+            });
+            trackError(error, Submit.AuthMockNumbersUpdate);
+        }
+    }
+
+    const addPhoneNumber = (number?: MockNumber) => {
```

Comment:
```suggestion
    function addPhoneNumber(number?: MockNumber) {
```
we usually stick to functions for this 👍🏻